capstone/src/main/java/com/google/sps/servlets/UpdateAssociationServlet.java

```diff
@@ -19,11 +22,25 @@
 @WebServlet("/update-associations")
 public class UpdateAssociationServlet extends HttpServlet {
 
-  public static final String SURVEY_ENTITY_KIND = "Actual";
+  public static final String SURVEY_ENTITY_KIND = "Response";
   public static final String COMMENT_PROPERTY = "text";
+  public static final String ZIPCODE = "zipcode";
 
   private LanguageServiceClient nlpClient;
   private DatastoreService datastore;
+  private static final List<String> SCOPES =
```

Comment:
You could call the MapData's precinctZipMap variable and load the keyset as the array of precincts

Reply:
Thanks for the suggestion! That's a really good idea, is that variable public? I guess one concern is mutability side effects since I need to add SF as an additional scope.

Reply:
Made this change on the earlier PR